src/box/lua/feedback_daemon.lua

```diff
@@ -26,13 +26,40 @@ local function get_fiber_id(f)
     return fid
 end
 
-local function fill_in_feedback(feedback)
+local function detect_docker_environment()
+    local fh = fio.open('/proc/1/cgroup', {'O_RDONLY'})
```

Comment:
But you could just check `/.dockerenv` file existence.

Reply:
I suppose, this file may be removed any time soon. It was originally used by lxc execution driver, which was removed from docker 4 years ago. See more [here](https://superuser.com/a/1021925). I would rather stick with "reading /proc/1/cgroup file" approach